vulcan/lib/server/controllers/workflows_controller.rb

```diff
@@ -11,5 +11,17 @@ def fetch
         symbolize_names: true)
     })
   end
+
```

Comment:
Note, I basically expect to throw this controller out, after we hook into Archimedes.